css/style.css

```diff
@@ -0,0 +1,589 @@
+  width: 177px;
+}
+
+#footer-content {
+  flex-direction: column;
+}
+
+#footer-logo {
+  width: 251px;
+}
+
+.fa {
+  color: #656665;
+}
+
+#nl-img-cont {
+  height: 180px;
+}
+
+#newsletter-large-pic {
+  display: none;
+}
+
+div.smalldiv {
+  border: none !important;
+}
+
+@media screen and (max-width: 768px) and (min-width: 426px) {
+  .navbar-toggler-icon {
+    display: inline;
+  }
+
+  .feature-section,
+  .story-section,
+  .debate-section {
+    height: auto;
+  }
+
+  .feature-section,
+  .more-stories1 {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 4rem;
+  }
+
+  .feature-cont {
+    align-items: flex-start;
+  }
+
+  .image-thumbnail1 {
+    margin-left: auto;
+  }
+
+  .last-section {
+    flex-direction: row;
+  }
+
+  .email-subscription-top {
+    width: 40%;
+    height: auto;
+    display: inline;
+  }
+
+  .email-subscription {
+    width: 100%;
+    height: auto;
+  }
+
+  .right-news-cont {
+    display: none;
+  }
+
+  .more-stories1 {
+    position: -webkit-sticky;
+    position: sticky;
+    bottom: 1rem;
+    align-self: flex-end;
+  }
+
+  #newsletter-large-pic {
+    display: inline;
+    background: url("https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&h=265&q=90&f=742bce9a6f72dd8f19ba2e7fd5db06e5");
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 254px;
+  }
+
+  #sticky-footer {
+    margin: 0;
+    padding: 0;
+  }
+
+  .footer-copyright,
+  .social {
+    align-self: flex-end;
+    margin-right: auto;
+  }
+
+  #footer-content {
+    flex-direction: row;
+  }
+
+  .footer-copyright {
+    align-self: flex-end;
+  }
+}
+
+@media screen and (max-width: 991px) {
+  #middle-navbar,
+  nav.nav-navigation,
+  nav.navia {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  main {
+    margin-top: 20px;
+  }
+
+  nav.navia {
+    position: relative;
+    z-index: -1;
+  }
+
+  header.header-container {
+    position: -webkit-sticky; /* Safari */
+    position: sticky;
+    top: 0;
+  }
+
+  .top-header {
+    padding: 0;
+  }
+
+  #middle-navbar {
+    display: block;
+  }
+
+  .weather-icon {
+    width: 40px;
+  }
+
+  picture {
+    display: inherit;
+  }
+
+  /* This is for the middle navbar */
+
+  .nav-navigation {
+    display: flex;
+    justify-content: space-between;
+    height: 55px;
+  }
+
+  .navbar-nav1 {
+    display: flex;
+    flex-wrap: nowrap;
+    align-content: center;
+    align-items: stretch;
+    width: 90%;
+    height: 54px;
+  }
+
+  .navbar-nav1 li {
+    list-style-type: none;
+    padding: 3px;
+    flex-grow: 1;
+  }
+
+  ul.navbar-nav1 {
+    font-size: 1rem;
+    line-height: 24px;
+    font-weight: bold;
+    font-family: "Roboto Condensed", sans-serif;
+    color: black;
+  }
+
+  ul.navbar-nav1 li a {
+    color: black;
+    display: block;
+    padding: 15px 5px;
+    flex-grow: 1;
+  }
+
+  #nav-link-long {
+    padding-left: 0;
+    padding-right: 0;
+  }
```

Comment:
- [ ] There is a bootstrap class for this behavior, please use bootstrap whenever it is possible.